specificatie/verblijfplaats.yaml

```diff
@@ -271,59 +241,27 @@ components:
               $ref: '#/components/schemas/AdresseerbaarObjectIdentificatie'
             nummeraanduidingIdentificatie:
               $ref: '#/components/schemas/NummeraanduidingIdentificatie'
-            adresregel1:
-              $ref: '#/components/schemas/Adresregel1'
-            adresregel2:
-              $ref: '#/components/schemas/Adresregel2'
             woonplaats:
               $ref: '#/components/schemas/Woonplaats'
-            indicatieVestigingVanuitBuitenland:
-              $ref: '#/components/schemas/IndicatieVestigingVanuitBuitenland'
             korteNaam:
               $ref: '#/components/schemas/KorteNaam'
-            vanuitVerblijfplaatsOnbekend:
-              $ref: '#/components/schemas/VanuitVerblijfplaatsOnbekend'
             datumVan:
               $ref: 'datum.yaml#/components/schemas/AbstractDatum'
             datumIngangGeldigheid:
               $ref: 'datum.yaml#/components/schemas/AbstractDatum'
-            datumInschrijvingInGemeente:
-              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
-            datumVestigingInNederland:
-              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
-            gemeenteVanInschrijving:
-              $ref: "common.yaml#/components/schemas/Waardetabel"
-            landVanwaarIngeschreven:
-              $ref: 'common.yaml#/components/schemas/Waardetabel'
             inOnderzoek:
               $ref: "#/components/schemas/AdresInOnderzoek"
     Locatie:
       allOf:
         - $ref: '#/components/schemas/AbstractVerblijfplaats'
         - type: object
           properties:
-            adresregel1:
-              $ref: '#/components/schemas/Adresregel1'
-            adresregel2:
-              $ref: '#/components/schemas/Adresregel2'
-            datumInschrijvingInGemeente:
-              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
-            datumVestigingInNederland:
-              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
             datumVan:
               $ref: 'datum.yaml#/components/schemas/AbstractDatum'
             functieAdres:
               $ref: 'common.yaml#/components/schemas/Waardetabel'
-            gemeenteVanInschrijving:
-              $ref: "common.yaml#/components/schemas/Waardetabel"
-            indicatieVestigingVanuitBuitenland:
-              $ref: '#/components/schemas/IndicatieVestigingVanuitBuitenland'
-            landVanwaarIngeschreven:
-              $ref: 'common.yaml#/components/schemas/Waardetabel'
             locatiebeschrijving:
               $ref: '#/components/schemas/Locatiebeschrijving'
-            vanuitVerblijfplaatsOnbekend:
-              $ref: '#/components/schemas/VanuitVerblijfplaatsOnbekend'
             woonplaats:
```

Comment:
woonplaats moet uit Locatie verwijderd worden. Deze kan nooit een waarde hebben, aangezien we die niet meer samenstellen.

Reply:
woonplaats verwijderd.